examples/_quick-start/python/example.py

```diff
@@ -8,9 +8,10 @@
 client = hvac.Client(
     url='http://127.0.0.1:8200',
     token='dev-only-token',
+    namespace='admin',  # If namespace is configured, make sure to set here
 )
 
-# Writing a secret
+# Writing a secret in the default secret engine
```

Comment:
what do you mean "in the default secret engine"? this is the kv-v2 engine

Reply:
So to my understanding if no engine is specified, then kv-v2 engine is used. I didn't want to call out the  engine by name just in case the default engine in the future changes. Honestly,  this comment can be removed.